kustomize/commands/internal/util/util_test.go

```diff
@@ -95,6 +95,47 @@ func TestGlobPatternsWithLoaderRemoteFile(t *testing.T) {
 	}
 }
 
+func TestIsHiddenFilePath(t *testing.T) {
+	hiddenFilePaths := []string{
```

Comment:
suggestion: in addition to the glob expressions, include test data that looks like full file paths, since we use this function with both types of input. It might make sense to use a table test instead of multiple loops, e.g.:

```go
tests := map[string]struct {
		paths        []string
		expectHidden bool
	}{
		"hidden globs": {
			expectHidden: true,
			paths: []string{
				".*",
				"/.*",
				"dir/.*"},
		},
               // more categories of examples
	}
	for name, test := range tests {
		test := test
		t.Run(name, func(tt *testing.T) {
			for _, path := range test.paths {
				require.Equal(tt, test.expectHidden, IsHiddenFilePath(path), "path %s not correctly identified", path)
			}
		})
	}
}
```